commands/conversions/create-gif-from-video.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+# Dependency: This script requires `gifski` to be installed: https://gif.ski/
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Create GIF from video
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.icon 📹
+# @raycast.argument1 { "type": "text", "placeholder": "Path to the source file (default: take the last screen recording on the Desktop)", "optional": true }
+# @raycast.argument2 { "type": "text", "placeholder": "FPS (between 1 and 60, default: 20)", "optional": true }
+# @raycast.argument3 { "type": "text", "placeholder": "Width (default 600)", "optional": true }
+# @raycast.packageName Utilities
+
+# Documentation:
+# @raycast.description Create a GIF from video, by default it takes the last screen record video
+# @raycast.author Quentin Eude
+# @raycast.authorURL https://github.com/qeude
+
+import sys
+import mimetypes
+from glob import glob
+import os
+from shutil import which
+import subprocess
+from pathlib import Path
```

Comment:
A small suggestion just to organize
```suggestion
import mimetypes
import os
import subprocess
import sys

from glob import glob
from pathlib import Path
from shutil import which
```